server/swap/swap.go

```diff
@@ -343,18 +342,20 @@ out:
 }
 
 func (s *Swapper) tryConfirmSwap(status *swapStatus) {
-	if status.swapTime != zeroTime && status.swapConfirmed == zeroTime {
-		confs, err := status.swap.Confirmations()
-		if err != nil {
-			// The transaction has become invalid. No reason to do anything.
-			return
-		}
-		// If a swapStatus was created, the asset.Asset is already known to be in
-		// the map.
-		if confs >= int64(s.coins[status.swapAsset].SwapConf) {
-			status.swapConfirmed = time.Now()
-		}
+	if status.swapTime.IsZero() || !status.swapConfirmed.IsZero() {
+		return
 	}
```

Comment:
Reducing the average indentation of the function is just a preference of mine, and I'd gladly change it back if the readability is negatively affected.